src/common/darktable.c

```diff
@@ -112,6 +117,9 @@ static int usage(const char *argv0)
   printf("  --configdir <user config directory>\n");
   printf("  -d {all,cache,camctl,camsupport,control,dev,fswatch, input,lighttable,\n");
   printf("      lua, masks,memory,nan,opencl, perf,pwstorage,print,sql}\n");
+#ifdef _WIN32
+  printf("     Note: debug log will be written to this file: %s\n", logfile);
```

Comment:
That shouldn't be part of the `-d` option as it affects all output, not just the debug log.

Reply:
I can move it to the end of the usage text, no problem.